package.json

```diff
@@ -30,11 +31,11 @@
   "devDependencies": {
     "execa": "^4.1.0",
     "fast-glob": "^3.3.2",
-    "ic-mops": "^1.1.1",
+    "ic-mops": "^1.4.0",
```

Comment:
Newest mops 1.4.0 is needed. [Here](https://github.com/ZenVoich/mops/issues/268#issuecomment-2743108161) is why.